README.md

```diff
@@ -1,50 +1,166 @@
-# The Lassie Lounge URL Shortener 
 
-Hacktoberfest Project
+<div align="center">
+  <img src="https://raw.githubusercontent.com/webdev-20/shorti-url-shortener/2a16fa1e8d6a13c4e6aacd5ebd517ed1b6795001/client/public/images/shorti.svg">
+  <h3 align="center">
+      <a href="https://shorti.onrender.com" class="default">View Demo</a> · 
+      <a href="https://github.com/webdev-20/shorti-url-shortener/issues" class="default">Report Bug</a> · 
+      <a href="https://github.com/webdev-20/shorti-url-shortener/issues" class="default">Request Feature</a>
+  </h3>
+  <div>
 
-# The project
+  <a href="">[![Issues](https://img.shields.io/github/issues/webdev-20/shorti-url-shortener.svg?style=for-the-badge)](https://github.com/webdev-20/shorti-url-shortener/issues)</a>
+  <a href="">[![contributors](https://img.shields.io/github/contributors/webdev-20/shorti-url-shortener.svg?style=for-the-badge)](https://github.com/webdev-20/shorti-url-shortener/contributors)</a>
+  <a href="">[![Forked](https://img.shields.io/github/forks/webdev-20/shorti-url-shortener.svg?style=for-the-badge)](https://github.com/webdev-20/shorti-url-shortener)</a>
 
-URL shortener 
+  </div>
+</div>
 
-## Tech Stack
 
-- React
-- CSS Modules
-- Mongo
-- Express
+# The Lassie Lounge URL Shortner
```

Comment:
the title can be "Shorti URL Shortener", we didn't have a name

Reply:
I think it might be ok if we just remove it as the logo clearly says "Short URL Shortener" already